MkDocs/monitor_csv_importer_8.6/docs/prereqs.md

```diff
@@ -30,7 +30,7 @@ Monitor API credentials:
 Monitor DB2 Warehouse credentials:
 
 1.  Hostname
-2.  Database, usually BLUDB
+2.  Database, usually `BLUDB`
```

Comment:
Explain how they can find this incase it isn't BLUDB.

Reply:
I decided to remove ", usually `BLUDB`" as I have written the credentials information must be obtained from the Exercise facilitator or the MAS Administrator above.